index.html

```diff
@@ -256,7 +256,32 @@ <h3 class="connect-b">LET'S CONNECT</h3>
             </div>
         </article>
       </section>
-      
+
     </div>
+    <section class="form">
+      <div class="form_container">
+          <h4 class="form_heading">
+            Contact me
+          </h4>
+          <p class="p">If you have an application you are interested in developing, a feature that you need to build or a project that needs
+          coding, i'd love to help width it!</p>
+
+          <div class="form-wrap">
+            <form action="https://formspree.io/f/xpzbqzab" id="form-email" method="post">
+              <input type="text" name="name" id="name" placeholder="Full name" required maxlength="30">
+              <input type="email" name="email" id="email" placeholder="Alexandra@gmail.com" required>
+              <textarea name="msg" id="msg" class="textarea" cols="30" rows="10" placeholder="Write your message here..." maxlength="500"></textarea> 
```

Comment:
- 🙏Please add the `required` validation in the textarea. This will help prevent the field from being submitted without any content.
```suggestion
              <textarea name="msg" id="msg" class="textarea" cols="30" rows="10" placeholder="Write your message here..." maxlength="500" required></textarea>

```